src/libsync/CMakeLists.txt

```diff
@@ -68,22 +68,26 @@ else()
 endif()
 
 # These headers are installed for libowncloudsync to be used by 3rd party apps
-set(owncloudsync_HEADERS
-    ${CMAKE_CURRENT_BINARY_DIR}/owncloudlib.h
-    logger.h
-    accessmanager.h
-)
-
 INSTALL(
-    FILES ${owncloudsync_HEADERS}
-    DESTINATION ${INCLUDE_INSTALL_DIR}/${synclib_NAME}
+    FILES
+        ${CMAKE_CURRENT_BINARY_DIR}/owncloudlib.h
+        logger.h
+        accessmanager.h
+    DESTINATION ${INCLUDE_INSTALL_DIR}/ownCloud/libsync
 )
 
 add_library(libsync SHARED ${libsync_SRCS})
-
-target_link_libraries(libsync PUBLIC
-    csync
-    Qt5::Core Qt5::Network
+add_library(ownCloud::SyncCore ALIAS libsync)
+set_target_properties(libsync PROPERTIES EXPORT_NAME SyncCore)
```

Comment:
Perhaps just rename `libsync` entirely?

Reply:
We can now use ownCloud::SyncCore in our code

Reply:
What I mean is, we probably should just rename directories etc. in our code base, too, and use one name everywhere.

Reply:
If this is the plan, a TODO should be left above this code.